src/hipscat_import/soap/resume_plan.py

```diff
@@ -62,6 +62,7 @@ def gather_plan(self, args):
     def wait_for_counting(self, futures):
         """Wait for counting stage futures to complete."""
         self.wait_for_futures(futures, self.COUNTING_STAGE)
+        self.touch_stage_done_file(self.COUNTING_STAGE)
```

Comment:
Anything you can check here in terms of unfinished work?

Reply:
Yes! Thanks!